### PR TITLE
Frontend: added completionReadyEditor

### PR DESCRIPTION
**Pull request by kubaflo**
https://user-images.githubusercontent.com/42434498/236899850-d71dccf6-1bff-4df2-9d98-978926ca6344.mp4

### CODE REVIEW

File: src/Frontend/EventPage.xaml.cs [hunk 1]
```diff
@@ -135,6 +141,22 @@ void SaveClicked(object sender, EventArgs evArgs)
...
+            completionWords = JsonSerializer.Deserialize<HashSet<string>>(Preferences.Get("completionWords", null));
+
+        competionReadyEditor.AutocompletedWords = completionWords.ToList();
+    }
+
+    void SaveWords()
+    {
+        completionWords.UnionWith(competionReadyEditor.Text.Split(' ').ToHashSet());
+        Preferences.Set("completionWords", JsonSerializer.Serialize(completionWords));
+    }
```
Comment (knocte) [inline src/Frontend/EventPage.xaml.cs:161]: @kubaflo please stick to the current method of the app to save JSON content, don't invent new method

Comment (knocte) [reply src/Frontend/EventPage.xaml.cs:161]: (to save and to load) BTW did you test with words-with-dashes?

Comment (kubaflo) [reply src/Frontend/EventPage.xaml.cs:161]: @knocte  Simulator Screen Shot - iPhone 14 Pro Max - 2023-05-09 at 11 51 07

**Comment by kubaflo** [comment]
> One last night: can you make all suggestions be lower-case please (even if they come from a word that starts with uppercase).

@knocte done

**Comment by siwatanejo** [comment]
Superseded by https://github.com/siwatanejo/DG/pull/1

**Pull request closed without merging**